ViAn/GUI/videowidget.cpp

```diff
@@ -158,7 +177,13 @@ void VideoWidget::init_frame_processor() {
     f_processor = new FrameProcessor(&new_frame, &settings_changed, &z_settings, &video_width,
                                      &video_height, &new_frame_video, &m_settings, &v_sync, &frame_index, &o_settings, &overlay_changed);
 
-    QThread* processing_thread = new QThread();
+    try {
+        processing_thread = new QThread(this);
```

Comment:
Shouldn't the following block of code (line 187-212) also be inside the try block? There is no point in connecting if the thread cannot be created